docs/csharp/programming-guide/inside-a-program/coding-conventions.md

```diff
@@ -1,99 +1,205 @@
 ---
 title: "C# Coding Conventions - C# Programming Guide"
 description: Learn about coding conventions in C#. Coding conventions create a consistent look to the code and facilitate copying, changing, and maintaining the code.
-ms.date: 03/31/2021
-helpviewer_keywords: 
+ms.date: 05/05/2021
+helpviewer_keywords:
   - "coding conventions, C#"
   - "Visual C#, coding conventions"
   - "C# language, coding conventions"
 ---
+
 # C# Coding Conventions (C# Programming Guide)
 
-Coding conventions serve the following purposes:  
-  
-- They create a consistent look to the code, so that readers can focus on content, not layout.  
-  
-- They enable readers to understand the code more quickly by making assumptions based on previous experience.  
-  
-- They facilitate copying, changing, and maintaining the code.  
-  
-- They demonstrate C# best practices.  
-
-The guidelines in this article are used by Microsoft to develop samples and documentation.  
-  
-## Naming conventions  
-  
-- In short examples that don't include [using directives](../../language-reference/keywords/using-directive.md), use namespace qualifications. If you know that a namespace is imported by default in a project, you don't have to fully qualify the names from that namespace. Qualified names can be broken after a dot (.) if they are too long for a single line, as shown in the following example.
+Coding conventions serve the following purposes:
+
+- They create a consistent look to the code, so that readers can focus on content, not layout.
+- They enable readers to understand the code more quickly by making assumptions based on previous experience.
+- They facilitate copying, changing, and maintaining the code.
+- They demonstrate C# best practices.
```

Comment:
```suggestion
> [!div class="checklist"]
>
> - They create a consistent look to the code, so that readers can focus on content, not layout.
> - They enable readers to understand the code more quickly by making assumptions based on previous experience.
> - They facilitate copying, changing, and maintaining the code.
> - They demonstrate C# best practices.
```